CS342MemberViews/Sources/SherryXie/SherryPageModel.swift

```diff
@@ -0,0 +1,26 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+
+struct Page: Identifiable, Equatable {
```

Comment:
Really great job structuring your models independent of you views, a great job! 👍